packages/maskbook/src/extension/options-page/DashboardDialogs/Wallet/TransferTab.tsx

```diff
@@ -76,6 +98,14 @@ export function TransferTab(props: TransferTabProps) {
     }, [transferCallback])
     //#endregion
 
+    const ETH_ADDRESS = useConstant(CONSTANTS, 'ETH_ADDRESS')
+
+    const onClick = useCallback(async () => {
+        const fee = await getFee(chainId, account, ETH_ADDRESS, tokenBalance)
+        const balance = new BigNumber(Web3Utils.toWei(formatBalance(tokenBalance, token.decimals), 'ether')).minus(fee)
```

Comment:
FYI:

+ `tokenBalance` is an amount in WEI.
+ `fee` is an amount in WEI.
+ `amount` is an amount in WEI.

And you haven't checked if the `tokenBalance` is less than `fee`.

Reply:
why resolved?

Reply:
IDK